l3build-stdmain.lua

```diff
@@ -179,7 +179,7 @@ function main(target,names)
     errorlevel = target_list[target].func(names)
   end
   -- All done, finish up
-  if errorlevel ~= 0 then
+  if errorlevel or errorlevel ~= 0 then
```

Comment:
`s/or/and/`

Reply:
Perhaps I should close this PR, and suggest `l3build` to add words to the doc for `target_list`, to mention that the `func` should return 0 when successes.
https://github.com/latex3/l3build/blob/177e77d514df4fc139a47e889344833ebda0645e/l3build.dtx#L1931-L1934

Reply:
Is that the same as this https://github.com/latex3/l3build/issues/205?

Reply:
https://github.com/latex3/l3build/issues/205#issuecomment-977680131 makes me re-check the latest manual, and this time I find the line 
https://github.com/latex3/l3build/blob/177e77d514df4fc139a47e889344833ebda0645e/l3build.dtx#L1946

So it's totally my careless. I'm closing this PR.